stories/design-primitives/colours/ColourCard.tsx

```diff
@@ -23,7 +23,21 @@ type Props = {
   variableName: string;
 };
 
-const ColourCard: FunctionComponent<Props> = (props) => {
+const ColourCard = (props: Props) => {
```

Comment:
Shouldn't we just stick with `color`?

Reply:
Ha! I was hesitant to go broader with renaming 😄 

I pushed another commit, where I renamed all variables and files in this directory (and renamed the directory too). I left the 'u" in a couple of labels that appear on the screen, because this is the only bit that might matter to the viewer.